libcudacxx/share/libcudacxx/lldb/annotated_ptr.py

```
@@ -0,0 +1,60 @@
+    ptr_display = "nullptr"
+    try:
+        value_nonsynth = value.GetNonSyntheticValue()
+        repr_ptr = value_nonsynth.GetChildMemberWithName("__repr")
+        if repr_ptr.IsValid():
+            ptr_value = repr_ptr.GetValueAsUnsigned(0)
+            if ptr_value == 0:
+                ptr_display = "nullptr"
+            else:
+                ptr_display = f"{ptr_value:#x}"
+    except Exception:
+        pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)